keyvi/src/cpp/dictionary/fsa/automata.h

```diff
@@ -40,6 +40,7 @@
 #include "dictionary/util/endian.h"
 #include "dictionary/fsa/internal/intrinsics.h"
 #include "dictionary/dictionary_merger_fwd.h"
+#include "dictionary/util/keyvi_file.h"
```

Comment:
Yes, and with this class I wanna introduce that concept, and in future move file writing, and all truncation and other checks here.